_data/remote.yml

```diff
@@ -15,6 +15,14 @@ websites:
       software: Yes
       doc: https://secure.logmein.com/welcome/webhelp/EN/Pro/LogMeIn/c_common_Security_TwoStep.html
 
+    - name: LabTech Control Center
+      url: http://www.labtechsoftware.com/
+      img: labtech.png
+      tfa: Yes
+      software: Yes
+      hardware: Yes
```

Comment:
I can't find anything relating to hardware based authentication on the doc page or on AuthAnvil's official page.
Maybe you could point me in the right direction?